sdk/communication/communication-phone-numbers/samples-dev/getPurchasedPhoneNumber.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Demonstrates how to us the PhoneNumbersClient to get a purchased phone number.
+ */
+
+import { PhoneNumbersClient } from "@azure/communication-phone-numbers";
+
+export async function main() {
+  console.log("\n== Get a Purchased Phone Number Javascript Sample ==\n");
+
+  // You will need to set this environment variable or edit the following values
+  const connectionString =
+    process.env.COMMUNICATION_CONNECTION_STRING ||
+    "endpoint=https://resourceName.communication.azure.net/;accessKey=test-key";
+
+  try {
+    // create new client
+    const client = new PhoneNumbersClient(connectionString);
+
+    const phoneNumberToGet = "+16135550147";
```

Comment:
Where does this number come from? If it's a magic number that belongs to our static test resource, should we make it an environment variable?

Reply:
i'm curious about this as well, i want to migrate the SMS samples but the lack of an actual number could be an issue

Reply:
I'll make it a env variable. One of the reasons the samples fail in the pipeline is because an invalid string is passed.